fixtures/manufacturers.json

```diff
@@ -317,6 +317,11 @@
     "name": "LiteGear",
     "website": "https://www.litegear.com/"
   },
+  "litepanel": {
+    "name": "LitePanel",
+    "comment": "1 x 1 Gemini panel",
+    "website": "https://www.litepanels.com/en/litepanels/gemini-1x1-soft-rgbww-led-panel-(standard-yoke%2C-uk-power-cable)/c-1027/p-1811"
```

Comment:
This file is a database of the manufacturer only, so the fixture-specific data shouldn't be here.